specification/redis/resource-manager/Microsoft.Cache/preview/2019-07-01/examples/RedisCacheCreate.json

```diff
@@ -17,12 +17,11 @@
         },
         "enableNonSslPort": true,
         "shardCount": 2,
-        "replicasPerMaster": 2,
```

Comment:
Was this removed because the combination of shardCount = 2 and replicasPerMaster = 2 does not actually work? If so you might want to change it to replicasPerMaster =1 for the meantime. I think that should work.

Reply:
It will raise an error: ` (BadRequest) Clustering is not yet supported on a cache with replicas per master`

Reply:
OK - thanks for checking, I must have remembered wrongly!